### PR TITLE
readds armor variance, does FF changes

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
title
literally the title
.4/.6/.8
40/45/50 melee/acid
pas 11 has more ff protection

## Why It's Good For The Game
Because people said they wanted it back 
FF protection

## Changelog
:cl:
balance: readded armor variance to jaeger armor. Added FF changes. More bullet armor overall in armor.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -26. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by RenGusta** [comment]
No data, stinky

**Comment by Kuro-ai** [comment]
Ah boy here we go again

**Comment by mrhugo13** [comment]
My main problem with the main PR rework was this problem in particular, nothing worse then 3 options that all do the same thing. 
Thankfully the PR fixes this which i like quite alot.

**Comment by Chaznoodles** [comment]
image

File: code/modules/modular_armor/armor_pieces.dm [hunk 1]
```diff
@@ -163,22 +171,22 @@
...
-	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
-	slowdown = 0.1
+	soft_armor = list("melee" = 5, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 5)
+	slowdown = 0.05
 
 /obj/item/armor_module/armor/arms/marine/assault
-	name = "\improper Jaeger Pattern Assault arm plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
+	name = "\improper Jaeger Pattern Heavy Assault arm plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
```
Comment (psykzz) [inline code/modules/modular_armor/armor_pieces.dm:189]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides higher protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
```

File: code/modules/modular_armor/armor_pieces.dm [hunk 2]
```diff
@@ -125,21 +129,25 @@
...
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides minor protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Skirmisher armor piece."
 	icon_state = "skirmisher_legs"
+	soft_armor = list("melee" = 5, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 5)
+	slowdown = 0.05
 
 /obj/item/armor_module/armor/legs/marine/assault
-	name = "\improper Jaeger Pattern Assault leg plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
+	name = "\improper Jaeger Pattern Heavy Assault leg plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
```
Comment (psykzz) [inline code/modules/modular_armor/armor_pieces.dm:147]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides higher protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
```

File: code/modules/modular_armor/armor_pieces.dm [hunk 3]
```diff
@@ -86,21 +86,25 @@
...
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides minor protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Skirmisher armor piece."
 	icon_state = "skirmisher_chest"
+	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 5)
+	slowdown = 0.1
 
 /obj/item/armor_module/armor/chest/marine/assault
-	name = "\improper Jaeger Pattern Assault chestplates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece.."
+	name = "\improper Jaeger Pattern Heavy Assault chestplates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece.."
```
Comment (psykzz) [inline code/modules/modular_armor/armor_pieces.dm:104]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides higher protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece.."
```

File: code/modules/modular_armor/armor_pieces.dm (hunk 1 above)
Comment (XSlayer300) [reply code/modules/modular_armor/armor_pieces.dm:189]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides higher protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a assault armor piece."
```

File: code/modules/modular_armor/armor_pieces.dm (hunk 2 above)
Comment (XSlayer300) [reply code/modules/modular_armor/armor_pieces.dm:147]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides higher protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a assault armor piece."
```

File: code/modules/modular_armor/armor_pieces.dm (hunk 3 above)
Comment (XSlayer300) [reply code/modules/modular_armor/armor_pieces.dm:104]:
```suggestion
	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a infantry armor piece."
```

Comment (Pariah919) [reply code/modules/modular_armor/armor_pieces.dm]: That doesn't make sense.
It's specifically minor/moderate/high.

**Comment by RenGusta** [comment]
To those on this PR like @mrhugo13 who think this is an improvement by virtue of variance, it's the same gaps in armor performance as was previously the case, without the benefit of huge durability at base line.

I did data for an armor variable stat set that was small but substantial but it was ignored in favour of the vast gaps in performance that this PR is trying to make. This will just take us back to where we were, with 'Light' only being used by the suicidal, and the 'Heavy' equivalent being overwhelmingly present.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-34). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**